File: src/ClientApp/src/app/pages/clients/dtos/update-client-contact.request.ts
export interface UpdateClientContactRequest {
    contactId: string;
    firstName: string;
    lastName: string;
    fullName: string;
    designation?: string | null;
    department?: string | null;
    email?: string | null;
    phone?: string | null;
    mobile?: string | null;
    status: number;
    notes?: string | null;
}

File: src/API/src/Modules/Clients/Clients.Application/DTOs/UpdateClientContactRequest.cs
using Clients.Domain.Enums;

namespace Clients.Application.DTOs
{
    public class UpdateClientContactRequest
    {
        public Guid ContactId { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string FullName { get; set; } = string.Empty;
        public string? Designation { get; set; }
        public string? Department { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Mobile { get; set; }
        public ClientContactStatus Status { get; set; } = ClientContactStatus.Active;
        public string? Notes { get; set; }
    }
}
